action.yml

```diff
@@ -36,6 +36,10 @@ inputs:
     description: "Url of the API where the application is running"
     required: false
     default: "https://optic-release-automation-ocrlhra4va-ue.a.run.app/"
+  sync-major:
```

Comment:
Can we also update the minor in case of a patch version?

Reply:
Yeah, on it. I'll change the name of the input to reflect what it does better